samples/suit/smp_transfer/README.rst

```diff
@@ -268,91 +277,205 @@ After programming the sample to your development kit and updating the sequence n
+                     signatureVerificationPolicy: signature verification on update and boot
+                     digest: 5dac29159363cbc50552a51ec8014d3927cbc854f83d8575af74ee27ee426c77
+                     digestAlgorithm: sha256
+                     signatureCheck: signature check passed
+                     sequenceNumber: 50331648
+                     semantic version: 3.0.0
+               role(31) (Radio Local Manifest)
+                     classId: 816aa0a0-af11-5ef2-858a-feb668b2e9c9 (nRF54H20_sample_rad)
+                     vendorId: 7617daa5-71fd-5a85-8f94-e28d735ce9f4 (nordicsemi.com)
+                     downgradePreventionPolicy: downgrade allowed
+                     independentUpdateabilityPolicy: independent update forbidden
+                     signatureVerificationPolicy: signature verification disabled
+                     digest: 5fb501225293ab78ec0caf14457543514550044757de2fa8a5e764ec059eae0d
+                     digestAlgorithm: sha256
+                     signatureCheck: signature check not performed
+                     sequenceNumber: 1
+                     semantic version: 0.1.0
+               role(20) (Root Manifest)
+                     classId: 3f6a3a4d-cdfa-58c5-acce-f9f584c41124 (nRF54H20_sample_root)
+                     vendorId: 7617daa5-71fd-5a85-8f94-e28d735ce9f4 (nordicsemi.com)
+                     downgradePreventionPolicy: downgrade allowed
+                     independentUpdateabilityPolicy: independent update allowed
+                     signatureVerificationPolicy: signature verification disabled
+                     digest: a0d99d177511ce908aaa7b74a8c595fc06060673976d09466b0d2c26f6547e2d
+                     digestAlgorithm: sha256
+                     signatureCheck: signature check not performed
+                     sequenceNumber: 1
+                     semantic version: 0.1.0
+               role(22) (Application Local Manifest)
+                     classId: 08c1b599-55e8-5fbc-9e76-7bc29ce1b04d (nRF54H20_sample_app)
+                     vendorId: 7617daa5-71fd-5a85-8f94-e28d735ce9f4 (nordicsemi.com)
+                     downgradePreventionPolicy: downgrade allowed
+                     independentUpdateabilityPolicy: independent update forbidden
+                     signatureVerificationPolicy: signature verification disabled
+                     digest: f391a4e298e0fc76d3b5d3044e379255484a2afde5e57d03dee5e07c9b4f558d
+                     digestAlgorithm: sha256
+                     signatureCheck: signature check not performed
+                     sequenceNumber: 1
+                     semantic version: 0.1.0
+
+         #. Upload the image with nrfutil:
 
             .. code-block:: console
 
-               mcumgr --conntype serial --connstring "dev=/dev/ttyACM0,baud=115200" image upload build/DFU/root.suit
+               nrfutil suit upload-envelope --serial-port=/dev/ttyACM0 --envelope-file build/DFU/root.suit
```

Comment:
Should we also show first to customers how to check COM port where DK is connected? I mean "nrfutil device list"

Reply:
Yes, will add it.